### ISSUE TITLE
Multiple bots

### ISSUE THREAD

**Issue by eden764** [finndev/PokeBuddy#521]

**Comment by curtiola** [comment]
Yes multiple instances of pokebuddy is possible, to have multiple ones with different settings just make copies of the whole pokebuddy-master directory

**Comment by curtiola** [comment]
Oh, sorry I just saw your title before you added the description, I've never tried more than 3

**Comment by TheUserCB** [comment]
The pokebuddy uses about 100mb of ram, then you need to have a good computer to use, here I have 16gb and normally use as you need.

**Comment by Kromes** [comment]
at 5 i got same problem .. they just walk and walk and did i said walk ?? :P 4 its max atm without any bug for me .. and i have a freaking good pc :P

**Comment by eden764** [comment]
@TheUserCB I got 16gb ram, i belive thats not the prob...
@Kromes yeah thats what happens to me after like 2 min that every thing runing good
